### PR TITLE
PCHR-798: Improve my leave block default absence period to the current one

### PR DESCRIPTION
**Pull request by omar-compucorp**
# Problem 

My leave block absence period selector was not pointing to the current period.

# Solution

I fixed it to set the default option to the current period based on the current date , and in case the current date do not fit into any absence period it will default to the latest period in term of End Date.

selection_002

selection_001

### CODE REVIEW

**Comment by AkA84** [comment]
I think you could squash those two commits together

File: civihr_employee_portal/views/includes/civihr_employee_portal_filter_absence_period.inc [hunk 1]
```diff
@@ -55,5 +55,26 @@ class civihr_employee_portal_filter_absence_period extends views_handler_filter_
       $this->options['expose']['label'] = '';
     }
   }
+
+  /**
+   * Get the default period id from a list of periods based on current time
+   */
```
Comment (AkA84) [inline civihr_employee_portal/views/includes/civihr_employee_portal_filter_absence_period.inc]: Please also document the parameters and return values

**Comment by coldrunKacper** [comment]
Yup, please document method's parameters and return values. Everything else looks fine.

**Comment by coldrunKacper** [comment]
Approved.

**Pull request merged**